tests/test_vlan.py

```diff
@@ -12,6 +12,80 @@ def check_syslog(self, dvs, marker, process, err_log, vlan_str, expected_cnt):
         (exitcode, num) = dvs.runcmd(['sh', '-c', "awk \'/%s/,ENDFILE {print;}\' /var/log/syslog | grep %s | grep \"%s\" | grep -i %s | wc -l" % (marker, process, err_log, vlan_str)])
         assert num.strip() == str(expected_cnt)
 
+    def create_acl_table(self, dvs, table_name, ports):
```

Comment:
I think it would be better to move these functions to a new file like test_acl_vlan.py. So that we do all ACL related test cases there.  I suggest we add one more test case where we create VLAN and ACL table after we removed VLAN and ACL Table in the first test case.

Reply:
It‘s a good idea, I'll do it in the next commit.Thank you!